lib/DataSet.js

```diff
@@ -122,7 +117,7 @@ DataSet.prototype.setOptions = function(options) {
 
 /**
  * Subscribe to an event, add an event listener
- * @param {String} event        Event name. Available events: 'put', 'update',
+ * @param {String} event        Event name. Available events: 'add', 'update',
```

Comment:
Yesterday, I learned this should be `{string}`. But I suppose that the other PR will handle that.

Reply:
Yes, to be addressed in a follow up PR.